src/Facade/ResourceBuilders/RequisitionResourceBuilder.cs

```diff
@@ -177,6 +177,28 @@ private IEnumerable<LinkResource> BuildLinks(RequisitionHeader model, IList<stri
                     yield return new LinkResource { Rel = "print-qc-labels", Href = href };
                 }
 
+                if (model.StoresFunction?.FunctionCode == "BOOKSU")
+                {
+                    var qc = model.Lines?.FirstOrDefault()?.TransactionDefinition.QcType;
+                    var printState = "PASS";
+                    if (qc == "Q")
+                    {
+                        printState = "QUARANTINE";
+                    }
```

Comment:
you could move this into a little function on the TransactionDefinition class? something like GetPrintState()